src/styles/Aside.css

```diff
@@ -13,7 +13,7 @@
   justify-content: space-around;
   cursor: pointer;
```

Comment:
also for such generic properties, use helper class

Reply:
Okay, I'll do that